modules/gapi/perf/cpu/gapi_core_perf_tests_cpu.cpp

```diff
@@ -311,6 +311,12 @@ INSTANTIATE_TEST_CASE_P(KMeans3DPerfTestCPU, KMeans3DPerfTest,
                                        cv::KMEANS_PP_CENTERS     | cv::KMEANS_USE_INITIAL_LABELS),
                                 Values(cv::compile_args(CORE_CPU))));
 
+INSTANTIATE_TEST_CASE_P(TransposePerfTestCPU, TransposePerfTest,
+                        Combine(Values(AbsExact().to_compare_f()),
+                                Values(szSmall128, szVGA, sz720p, sz1080p),
+                                Values(CV_8UC1, CV_16UC1, CV_16SC1, CV_32FC1),
```

Comment:
Are three-channel images supported? I believe it is the case for MTCNN (BGR), @dbudniko  please correct me.

Reply:
i've added additional types to param set and it still works OK